run-test.sh

```diff
@@ -29,7 +29,7 @@ usage()
     echo "                                      default: <repo_root>/bin/Product/<OS>.x64.<ConfigurationGroup>"
```

Comment:
I say we keep it if the cost to do so is not too high. We may need to use this for certain OSes where we have trouble building in the native environment, and have to build-then-zip from Windows.